<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="clear_screen_button">
       <property name="text">
        <string>Clear Screen</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="load_graph_button">
       <property name="text">
        <string>Load Graph</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>450</y>
      <width>371</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Utils</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="util_rotate_left_button">
         <property name="text">
          <string>Rotate Left</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="util_rotate_right_button">
         <property name="text">
          <string>Rotate Right</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="util_reset_rotation_button">
         <property name="text">
          <string>Reset Rotation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="util_reset_zoom_button">
         <property name="text">
          <string>Reset Zoom</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>101</width>
      <height>94</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Tools</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tool_zoom_button">
       <property name="text">
        <string>Zoom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tool_select_button">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>570</y>
      <width>371</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Start Point</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>End Point</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="1">
        <widget class="QLCDNumber" name="lcd_start_y"/>
       </item>
       <item row="2" column="1">
        <widget class="QLCDNumber" name="lcd_start_x"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLCDNumber" name="lcd_end_x"/>
       </item>
       <item row="0" column="3">
        <widget class="QLCDNumber" name="lcd_end_id"/>
       </item>
       <item row="3" column="3">
        <widget class="QLCDNumber" name="lcd_end_y"/>
       </item>
       <item row="0" column="1">
        <widget class="QLCDNumber" name="lcd_start_id">
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Filled</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>140</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Compute Algorithms</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="algo_bfs_button">
       <property name="text">
        <string>BFS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="algo_djikstra_button">
       <property name="text">
        <string>Djikstra</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="algo_astar_button">
       <property name="text">
        <string>A*</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>700</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="set_start_button">
       <property name="text">
        <string>Set Start Point</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="set_end_button">
       <property name="text">
        <string>Set End Point</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>97</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected Point:</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>610</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Visited Verts:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Distance:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdNumber_4"/>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdNumber_5"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>611</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>ID</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>X</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Y</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLCDNumber" name="lcd_selected_id"/>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_selected_x"/>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_selected_y"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1206</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMPF">
    <property name="title">
     <string>MPF</string>
    </property>
   </widget>
   <addaction name="menuMPF"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
